concreate/tools.py

```diff
@@ -16,10 +16,14 @@
 cfg = {}
 
 
-def parse_cfg():
+def init_cfg(config_file):
+    config_file = os.path.expanduser(config_file)
+    logger.info("Using concreate configuration file: %s" % (config_file))
     cp = configparser.ConfigParser()
-    cp.read(os.path.expanduser('~/.concreate'))
-    return cp._sections
+    cp.read(config_file)
+    global cfg
```

Comment:
this is not needed - modules can be used as "singletons"

Reply:
This supports --config=myconfig.ini

Reply:
global keyword is not required. you can do import tools. then tools.cfg is behaving like singleton.

Reply:
global is required as ``cfg`` is defined in this package (i.e. this is in the tools package itself).